components/scrollable-container/demo/demo.html

```diff
@@ -1,59 +1,83 @@
-<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <link rel="stylesheet" href="../coherent-gameface-components-theme.css">
     <link rel="stylesheet" href="../node_modules/coherent-gameface-slider/styles/horizontal.css">
     <link rel="stylesheet" href="../node_modules/coherent-gameface-slider/styles/vertical.css">
     <link rel="stylesheet" href="../style.css">
+    <style>
+        .dynamic-content {
+            display: none;
+        }
+
+        div.visible {
+            display: block !important;
+        }
```

Comment:
TODO to self: check

Reply:
I don't understand, what is self check?

Reply:
Tony set a reminder to check something on this line but seems that he didn't delete the comment :D

Reply:
I didn't like that it is a complex selector plus it has `!important`.
Why do we need this at all?